### PR TITLE
FIX #205: Problem importing mapKeys from lodash.

### LINKED ISSUE #205: Problem with lodash and persisting workflow state
I get this error from the persistence module.

```TypeError: (0 , lodash_mapkeys_1.default) is not a function
    at MongodbPersistence.saveWorkflowState (C:\_dev\experiment\substrate-relayer-tsoa\node_modules\@node-ts\bus-mongodb\dist\mongodb-persistence.js:76:60)
    at C:\_dev\experiment\substrate-relayer-tsoa\node_modules\@node-ts\bus-core\dist\workflow\registry\workflow-registry.js:152:44
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) undefined rejected {
  status: 'rejected',
  reason: TypeError: (0 , lodash_mapkeys_1.default) is not a function
      at MongodbPersistence.saveWorkflowState (C:\_dev\experiment\substrate-relayer-tsoa\node_modules\@node-ts\bus-mongodb\dist\mongodb-persistence.js:76:60)
      at C:\_dev\experiment\substrate-relayer-tsoa\node_modules\@node-ts\bus-core\dist\workflow\registry\workflow-registry.js:152:44
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
}
```

I took a look at the compiled mongo-persistence lodash_mapkeys_1.default is undefined but if I remove .default its defined and the code works.

is it a configuration on my side or is it the compiled code that has an issue?

thank you,
David L

### PR DESCRIPTION
**Pull request by manarhusrieh**
This is to solve the issue #205  when loading mapKeys function from lodash, It throws an error mentioned in the issue.

### CODE REVIEW

File: packages/bus-mongodb/src/mongodb-persistence.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { Message, MessageAttributes } from '@node-ts/bus-messages'
 import { Db, MongoClient } from 'mongodb'
 import { MongodbConfiguration } from './mongodb-configuration'
 import { WorkflowStateNotFound } from './error'
-import mapKeys from 'lodash.mapkeys'
+import _ from 'lodash'
```
Comment (adenhertog) [inline packages/bus-mongodb/src/mongodb-persistence.ts:13]: `lodash` isn't part of the dependencies, and it's quite a large package to pull in just for `mapKeys`.

Could this be replaced with a quick implementation instead? something like:

```suggestion

const mapKeys = (obj: any, fn: (key: string, value: any) => string) => {
  return Object.keys(obj).reduce((acc, oldKey) => {
    const newKey = fn(oldKey, obj[oldKey])
    acc[newKey] = obj[oldKey]
    return acc
  }, {} as Record<string, unknown>)
}
```

Comment (manarhusrieh) [reply packages/bus-mongodb/src/mongodb-persistence.ts:13]: Good point, I will apply it and push the solution again. 
Thanks

Comment (manarhusrieh) [reply packages/bus-mongodb/src/mongodb-persistence.ts]: @adenhertog I have applied the fix. Check again.

Comment (adenhertog) [reply packages/bus-mongodb/src/mongodb-persistence.ts]: looks great, nice one

**Pull request merged**